example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,27 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.signify.hue.reactivebleexample">
 
-    <!-- required for API 18 - 30 -->
-    <uses-permission
-            android:name="android.permission.BLUETOOTH"
-            android:maxSdkVersion="30" />
-    <uses-permission
-            android:name="android.permission.BLUETOOTH_ADMIN"
-            android:maxSdkVersion="30" />
-
-    <!-- required for API 23 - 30 -->
-    <uses-permission-sdk-23
-            android:name="android.permission.ACCESS_COARSE_LOCATION"
-            android:maxSdkVersion="30" />
-    <uses-permission-sdk-23
-            android:name="android.permission.ACCESS_FINE_LOCATION"
-            android:maxSdkVersion="30" />
-
-    <!-- API 31+ -->
-    <uses-permission android:name="android.permission.BLUETOOTH_CONNECT" />
-    <uses-permission
-            android:name="android.permission.BLUETOOTH_SCAN"
-            android:usesPermissionFlags="neverForLocation" />
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Is needed for debugging? I think you need to add it only to the debug manifest

Reply:
Yes, it is needed for running the example app in debug mode. But this is just the example app, so it won't impact the plugin's required permissions.